include/HUDL/HUDL.hpp

```diff
@@ -1,32 +1,38 @@
+#ifndef _HUDL_
+#define _HUDL_
 
+namespace IO = EVT::core::IO;
 
-
-
-namespace DEV = EVT::core::DEV;
-
-
-
+namespace HUDL {
 class HUDL {
-public: 
-    
-    // default constructor for HUDL class
-    HUDL();
-  
-    // writes data to the LCD to show on the screen 
-    void data_write(unsigned char d);
+public:
+  IO::GPIO &reg_select; // PA_3
+  IO::GPIO &reset;      // PB_3
+  IO::GPIO &cs;         // PB_12
+  IO::SPI &spi;
 
-    // writes commands to the LCD to conrol the ST7565
-    void comm_write(unsigned char d);
+  // default constructor for HUDL class
+  HUDL(IO::GPIO &reg_select, IO::GPIO &reset, IO::GPIO &cs, IO::SPI &spi);
```

Comment:
This & spacing doesn't look like it matches our formatting standards. This might indicate a problem with the GitHub Action and/or the .clang-format file. See what you can do to fix this.